docs/man-pages/condor_token_create.rst

```diff
@@ -0,0 +1,123 @@
+      
+
+*condor_token_create*
+======================
+
+given a password file, create an authentication token
+:index:`condor_token_create<single: condor_token_create; HTCondor commands>`\ :index:`condor_token_create command`
+
+Synopsis
+--------
+
+**condor_token_create** **-identity** *user@domain* [**-keyid** *keyid*]
+[**-authz** *authz* ...] [**-lifetime** *value*]
+[**-token** *filename*] [**-debug**]
+
+**condor_token_create** [**-help** ]
+
+Description
+-----------
+
+*condor_token_create* will read a HTCondor password file inside the
+*SEC_PASSWORD_DIRECTORY* and use it to create an authentication token.
+The authentication token may be subsequently used by clients to authenticate
+against a remote HTCondor server.  Tokens allow fine-grained authentication
```

Comment:
How about "Tokens allow fine-grained authentication as individual HTCondor users, as opposed to pool password, where anything in possession of the pool password will authenticate as the same user." ?

Reply:
Alternatively, the rationale for using TOKEN instead of PASSWORD can just be a link to the [token section of the manual](https://htcondor.readthedocs.io/en/latest/admin-manual/security.html#token-authentication).

Reply:
I'm a bit loath to put links into a man page; I'll clarify the text inline.